packages/boa/tests/fixtures/es-module-loaders/test-esm-loader-export-default.mjs

```diff
@@ -0,0 +1,9 @@
+import os from 'py:os';
+import { len, range } from 'py:builtins';
+import { strictEqual } from 'assert';
+import { add, subtract } from './test-esm-loader-math.mjs';
```

Comment:
Is this necessary to test for?

Reply:
Yes, I think it's in the change range too.